Remotion/Core/Extensions/Utilities/MemberAccessUtility.cs

```diff
@@ -54,7 +54,7 @@ public static object GetAttributeMemberValue (MemberInfo reflectionObject, Type
       return GetFieldOrPropertyValue (attributes[0], fieldOrProperty);
     }
 
-    public static MemberInfo GetFieldOrProperty (Type type, string fieldOrPropertyName, BindingFlags bindingFlags, bool throwExceptionIfNotFound)
+    public static MemberInfo? GetFieldOrProperty (Type type, string fieldOrPropertyName, BindingFlags bindingFlags, bool throwExceptionIfNotFound)

```

Comment:
@MichaelKetting `bindingFlags` unused. Might need cleanup too?

Reply:
is broken since revision 405 in 2004 and now located in Extensions assembly. no fix. no touch. please introduce aligators into moat on way out.